libs/chunking/chunking-core/src/main/kotlin/net/corda/chunking/Constants.kt

```diff
@@ -0,0 +1,13 @@
+package net.corda.chunking
+
+class Constants {
+    companion object {
+        const val KB = 1024
+        const val MB = 1024 * KB
+        const val CORDA_MESSAGE_OVERHEAD = 1024 * 10
```

Comment:
It might be more readable using constants defined above
```suggestion
        const val CORDA_MESSAGE_OVERHEAD = 10 * KB
```